projects/ngx-openlayers/src/lib/sources/imagearcgisrest.component.ts

```diff
@@ -12,25 +14,25 @@ import { SourceComponent } from './source.component';
   providers: [{ provide: SourceComponent, useExisting: forwardRef(() => SourceImageArcGISRestComponent) }],
 })
 export class SourceImageArcGISRestComponent extends SourceComponent implements OnInit, OnChanges {
-  instance: source.ImageArcGISRest;
+  instance: ImageArcGISRest;
 
   @Input() projection: ProjectionLike | string;
   @Input() url: string;
-  @Input() attributions: Attribution[];
+  @Input() attributions: AttributionLike[];
   @Input() crossOrigin?: string;
-  @Input() imageLoadFunction?: ImageLoadFunctionType;
+  @Input() imageLoadFunction?: LoadFunction;
   @Input() params?: { [k: string]: any };
   @Input() ratio = 1;
   @Input() resolutions?: number[];
-  @Input() logo?: string | olx.LogoOptions;
+  @Input() logo?: string; // (string | olx.LogoOptions);
```

Comment:
It seems that `logo` is no more a property of ImageArcGISRest

https://openlayers.org/en/latest/apidoc/module-ol_source_ImageArcGISRest.html